changelog.d/10413.feature

```diff
@@ -0,0 +1 @@
+Support for [MSC2285 (hidden read receipts)](https://github.com/matrix-org/matrix-doc/pull/2285).
```

Comment:
(don't mind this, I'm just applying this myself so you're mentioned in the changelog for this; feel free to add it yourself next time :slightly_smiling_face:)

```suggestion
Support for [MSC2285 (hidden read receipts)](https://github.com/matrix-org/matrix-doc/pull/2285). Contributed by @SimonBrandner.
```